src/test/java/de/fraunhofer/aisec/cpg/frontends/java/JavaLanguageFrontendTest.java

```diff
@@ -359,19 +359,26 @@ void testCast() throws Exception {
     assertNotNull(stmt);
 
     VariableDeclaration e = stmt.getSingleDeclarationAs(VariableDeclaration.class);
-    assertEquals(TypeParser.createFrom("ExtendedClass", true), e.getType());
+    // This test can be simplified once we solved the Issue with
```

Comment:
What issue do you mean?

Reply:
We still need to have a discussion on how to represent names, do we always use FQNs or can we store the SimpleName only? Currently, it is inconsistent. For example, this case of type apparently Used the simple name previously.

Reply:
I would actually propose to introduce a `Name` class, similar to the `Type` class, which can be used to actually compare types and internally has a representation of what that name is, i.e. its FQN and so on.